src/pytypes/tests/test_models.py

```diff
@@ -28,5 +28,41 @@ def test_model(self) -> None:
             NotificationCreate.parse_obj(missing_container)
 
 
+class TestScaleset(unittest.TestCase):
+    def test_scaleset_size_lt_0(self) -> None:
+        self.assertRaises(
```

Comment:
please use `with self.assertRaises(ValueError):` here.  See `test_alnum_filter.py` for an example.

Reply:
updated.